lib/capsule_crm/taggable.rb

```diff
@@ -3,9 +3,11 @@ module Taggable
     extend ActiveSupport::Concern
 
     def tags
-      CapsuleCRM::Connection.get(
+      tags = CapsuleCRM::Connection.get(
         "/api/#{api_singular_name}/#{id}/tag"
-      )['tags']['tag'].map { |item| CapsuleCRM::Tag.new(item) }
+      )['tags']['tag']
+      tags = [] << tags if tags.is_a? Hash
```

Comment:
Simply evident ... sorry for that :/
Do you prefer a new and clean pull request instead ? If not I'll commit your suggestion